### ISSUE TITLE
Windows UI (minimize, maximize and close)

### ISSUE THREAD

**Issue by Head** [OpenBazaar/OpenBazaar-Client#908]
It would now be great if it managed to follow the most basic Windows UI guideline for the Windows Version, which is the app-window minimize, maximize and close buttons being on the right side.

**Comment by AvatarX001** [comment]
Ditto. I don't even care if you disregard the windows UI-style for the buttons. Just put them on the right side where they belong. It is super annoying to use the app otherwise. Also, you do really want me to have to bring this up for the foreseeable future?

**Comment by pryds** [comment]
Low priority, agreed, but if something is done, I would think having native buttons would be the best option, since opinions differ, end everyone probably wants what they are used to. These coloured buttons without any symbols on them, until you hover the mouse over them, look like Mac inspired ones, and they are totally strange to me, and I'll never remember which colour does what (let alone telling the colours apart, being colour blind as is 8% of all males), however I DO like them on the left side like they are now, because that's what I'm used to.

TLDR: If it was up to me: Native buttons, if any change is to be made.

**Comment by morebrownies** [comment]
We may have to change around the header bar a bit to make this work well on windows since there will be many icons stacked on the right hand side and not much on the left side. I'll probably have to shift things around to make it work well.

**Comment by ghost** [comment]
A small fix needed:

Now when OB window is already maximized, "Maximize (+)" button keeps its title, but it should be renamed to "Restore".

**Comment by drwasho** [comment]
@morebrownies is woring on this too

**Comment by jjeffryes** [comment]
![image](https://cloud.githubusercontent.com/assets/1584275/15331276/8b77c618-1c2e-11e6-84da-11ed9b737239.png)

Details:
- [ ] if the platform is Windows, a Windows-style bar should be added to the top of the client, and the BrowserWindow height should be increased by the height of the bar (leaving the area available for the rest of the app the same). 
- [ ] when the Windows bar is showing, the "traffic light" buttons should not be visible. This can be done by adding display:none to the control group parent div they are in.
- [ ] the top bar should have the minimize, maximize, and close buttons
- [ ] the top bar should have a title of "OpenBazaar" and display the name of the route (Settings, Discover, etc.), or if it is on a user page, the user handle or guid.

**Comment by pryds** [comment]
The last comment only specifically mentions Windows. I'm not sure what the plans are for other systems - maybe you're already on top of it - but just to be clear: Please, don't keep the "traffic lights" on other systems, e.g. Linux. They are REALLY weird and very alienating to users. Window control buttons should be instantly recognizable to users.

I understand it if it is not possible to go completely native for some reason (?) and you don't want to emulate the UI of all kinds of systems, but maybe at least use what seems to be most common across systems: Symbols on the buttons are visible at all times, not just while hovering, and maybe get rid of the funky colours :-)

In my personal opinion, the above screenshot would fit nicely with at least the Linux distro that I'm running, if native is a no-go.

**Comment by jjeffryes** [comment]
this was completed
